src/testdir/test_signals.vim

```diff
@@ -153,8 +153,7 @@ func Test_deadly_signal_TERM()
   call assert_equal(['foo'], getline(1, '$'))
 
   let result = readfile('XautoOut')
-  call assert_match('VimLeavePre triggered', result[0])
-  call assert_match('VimLeave triggered', result[1])
+  call assert_equal(["VimLeavePre triggered", "VimLeave triggered"], result)
```

Comment:
I would add something like:

    call assert_equal(2, len(result))

Reply:
Adding that assert is not needed after my change, because
line 156 already checks that result contains 2 strings.